modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/index.js

```diff
@@ -57,6 +59,12 @@ const TeamMembers = () => {
 					</Button>
 				</div>
 			</div>
+			{subscriptionGroups && (
```

Comment:
We're already verifying if subscriptionGroups has a valid value on Layout before accessing this component
Check here https://github.com/pt-liferay-solutions/liferay-portal/blob/master/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/BaseLayout/index.js#L51

All the values provided by the useOutletContext are valid